src/scripts/install_machine.py

```diff
@@ -54,11 +54,35 @@ def get_graphics_capabilies(connection):
 
 
 def prepare_graphics_params(connection):
+    # Use ipv6 as workaround for https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1492621
+    spice_address = '::1'
+
+    # Check to make sure loopback has an IPv6 address assigned
+    # if not default back to IPv4
+    try:
+        if subprocess.run(['ip'], shell=True).returncode == 0:
+            ipv6_loopback = subprocess.check_output(['ip', '-6', 'addr', 'show', 'dev', 'lo'])
+
+            if not "lo:" in str(ipv6_loopback):
+                spice_address = '127.0.0.1'
+        elif subprocess.run(['ifconfig'], shell=True).returncode == 0:
+            ipv6_loopback = subprocess.check_output(['ifconfig', 'lo'])
```

Comment:
Ugh, `ifconfig` -- do you really need that for OpenSUSE? This is so old, I'd really rather not call that as well.